src/vs/workbench/contrib/notebook/browser/contrib/format/notebookDefaultFormatter.ts

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as nls from 'vs/nls';
+import { IWorkbenchContribution } from 'vs/workbench/common/contributions';
+import { IExtensionService } from 'vs/workbench/services/extensions/common/extensions';
+import { Disposable } from 'vs/base/common/lifecycle';
+
+
+export class NotebookDefaultFormatter extends Disposable implements IWorkbenchContribution {
+
+	static readonly ID = 'workbench.contrib.notebook.defaultFormatter';
+	static readonly configName = 'notebook.defaultFormatter';
+
+	static extensionIds: (string | null)[] = [];
+	static extensionItemLabels: string[] = [];
+	static extensionDescriptions: string[] = [];
```

Comment:
we can read from https://github.com/microsoft/vscode/blob/5e68ffd760038bb156894a87f896910be510accd/src/vs/workbench/contrib/format/browser/formatActionsMultiple.ts#L44